app/controllers/admin/split_assignments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::SplitAssignmentsController < AuthenticatedAdminController
+  VIEW_LIMIT = 1000
+
+  def index
+    split = Split.find(params.require(:split_id))
+    @split_name = split.name
+    @assignments = split.assignments.includes(visitor: { identifiers: :identifier_type }).order(updated_at: :desc).limit(VIEW_LIMIT)
```

Comment:
Someone might have a stronger opinion on whether an index is necessary here. I think it's optional, because we want to preserve insert/update efficiency, and this query is not in the critical path (it is loaded by users on demand)

Reply:
One thing that might make this slow on big assignment datasets is having an order by clause, which requires fetching the whole assignment set even in light of the limit clause. We could consider removing it. My guess is it'd make this very fast regardless of assignment count.

Reply:
I think that's reasonable. My only hesitation is our experience has been the assignments you care about most are the ones you manually assigned last week.  But I'm already leaning YAGNI - either the data set is skimmable or you need search, there's not a lot of value in the in-between states.

Reply:
Yeah, if we see this as a scrappy solution for low-scale assignments it makes sense, and we can iterate if we see value.

Reply:
Sorting in ruby after retrieval might be a nice feature for skimmability under 1k rows, though it might mislead people to think that absence of a sorted row means something.